public/src/cards.js

```diff
@@ -95,6 +95,8 @@ let events = {
       pack[name] = 1
     }
     App.update()
+    if (document.hidden && App.state.beep)
```

Comment:
woah, i did not know about `document.hasFocus`. regardless, the user could simply be looking away from his screen